docs/earthfile/earthfile.md

```diff
@@ -406,6 +406,10 @@ Instructs Earthly to not overwrite the file creation timestamps with a constant.
 
 Instructs Earthly to keep file ownership information. This applies only to the *artifact form* and has no effect otherwise.
 
+{% hint style='info' %}
+Note that one must include the flag in the corresponding `SAVE ARTIFACT --keep-own ...` command.
```

Comment:
Nit: We generally use "you" as a tone in other pages.

Also, added a clarification that this remark applies to certain COPY's only.


```suggestion
Note that you must include the flag in the corresponding `SAVE ARTIFACT --keep-own ...` command, if using *artifact form*.
```